src/Symfony/Bridge/Doctrine/Form/Type/DoctrineType.php

```diff
@@ -263,9 +263,11 @@ public function configureOptions(OptionsResolver $resolver)
     /**
      * Return the default loader object.
      *
+     * @param mixed QueryBuilder
```

Comment:
Shouldn't it be `@param mixed $queryBuilder` instead?

Reply:
Fixed, thanks!